### ISSUE TITLE
Change default and additional graph data line colours

### ISSUE THREAD

**Issue by phillipgardiner** [open-sdg/open-sdg#1099]
As a result of https://github.com/open-sdg/open-sdg/issues/1094 a knock of effect has been the appropriateness of the current default colours used for the graph data lines. 

To ensure that where possible colour contrast ratios continue to be adhered to both against a black and white background but ALSO against adjacent colours, and also work for Deuteranomaly, Protanomaly, Protanopia and Deuteranopia colour blindness. The following is proposed:

**White background:**
#004466 - default data line
#55a6e5
#8d4d57
#cd7a00
#993300
#339966
#9966cc

![Screenshot (139)](https://user-images.githubusercontent.com/60919820/108832039-dc983580-75c2-11eb-9a73-b342059e5f69.png)

**Black background:**
#ffdd00 - default data line
#55a6e5
#8d4d57
#cd7a00
#993300
#339966
#9966cc

![Screenshot (140)](https://user-images.githubusercontent.com/60919820/108832513-8e376680-75c3-11eb-9c45-9096dc5ca072.png)

On cycling through the available default colours, the next data lines would continue to be stripped versions of the same colours.

Please note the colours listed above are exactly the same apart from the default data line.

**Comment by brockfanning** [comment]
Will be fixed by #1103
